src/vm-vcpu/src/vm.rs

```diff
@@ -286,6 +307,11 @@ impl<EH: 'static + ExitHandler + Send> KvmVm<EH> {
         self.fd.clone()
     }
 
+    #[cfg(target_arch = "aarch64")]
+    fn gic(&self) -> &Gic {
```

Comment:
Right now it doesn't look like this can panic because we're creating the vcpus when calling new. Can we add a comment for now why this expect is ok?